src/hammer-vlsi/defaults.yml

```diff
@@ -316,14 +316,15 @@ vlsi.inputs:
     y: 0
     # pitch (float) - pitch of bumps in microns
     pitch: 0.0
-    # cell (str) - Name of bump cell
+    # cell (str) - Name of the default bump cell
     cell: ""
     # assignments - List of BumpAssignment structs. You must specify one of name or no_connect.
     # If both are specified the bump will be left unconnected
     #  - name (Optional[str]) - The name of the net being assigned to a bump
     #  - no_connect (Optional[bool]) - If True the bump will be present but unconnected
     #  - x (int) - The X coordinate of the bump assigned to this net
     #  - y (int) - The Y coordinate of the bump assigned to this net
+    #  - custom_cell (Optional[str[) - MACRO name of a custom bump cell in place of the default for this bump. You must also specify the LEF and GDS for this bump in extra_libraries.
```

Comment:
I think the second sentence is redundant but its worth being verbose :)

Reply:
well, I think it would be redundant if there was a way to cross-check this  with the provided LEFs before sending it off to the CAD tools.